airbyte-api/src/main/openapi/config.yaml

```diff
@@ -4729,6 +4782,31 @@ components:
           $ref: "#/components/schemas/ResourceId"
         workspaceId:
           $ref: "#/components/schemas/WorkspaceId"
+    AttemptId:
+      type: integer
+      format: int32
+    TemporalWorkflowId:
+      type: string
+    SetTemporalWorkflowInAttemptRequestBody:
+      type: object
+      required:
+        - jobId
+        - attemptId
+        - temporalWorkflowId
+      properties:
+        jobId:
+          $ref: "#/components/schemas/JobId"
+        attemptId:
+          $ref: "#/components/schemas/AttemptId"
+        temporalWorkflowId:
+          $ref: "#/components/schemas/TemporalWorkflowId"
+    InternalOperationResult:
```

Comment:
Curious, why is this an object if it only has one field?

Reply:
@xiaohansong this is from your branch, what do you think? Should we simplify this to a single field instead of an object?